src/sentry/api/bases/sentryapps.py

```diff
@@ -77,17 +80,17 @@ class SentryAppsPermission(SentryPermission):
         "POST": ("org:write", "org:admin"),
     }
 
-    def has_object_permission(self, request: Request, view, organization):
```

Comment:
SentryApps will be control silo, which means they won't have direct access to organizations.  the context here represents an RPC lookup of org details.